cmake/ThriftGenerate.cmake

```diff
@@ -84,7 +84,7 @@ add_custom_command(
     --gen "mstch_cpp2:include_prefix=${include_prefix},process_in_event_base,stack_arguments"
     --gen "py"
     --gen "java:hashcode"
-    --gen "go"
+    --gen "go:thrift_import=github.com/facebook/fbthrift/thrift/lib/go/thrift"
```

Comment:
why add the dependency when compile and generation source code？

Reply:
The generated code depends on it. It has a default value, but out of date.

Reply:
But we use glide to manage the dependency.

Reply:
It is not about the dependency management. I mean the configuration of **thrift_import** is explicitly insert into the generated code. like:
```import "github.com/facebook/fbthrift/thrift/lib/go/thrift"```

Reply:
@darionyaphet The default import path is _github.com/facebook/fbthrift-go_, which seems no longer exist. You could check on the output of `/path/to/the/fbthrift/generator --help`, i.e. `/opt/nebula/third-party/fbthrift/bin/thrift1 --help`